content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -266,9 +266,22 @@ This ensures that DaemonSet pods are never evicted due to these problems.
 
 ## Taint Nodes by Condition
 
-The node lifecycle controller automatically creates taints corresponding to
-Node conditions with `NoSchedule` effect.
-Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+The node controller automatically creates taints with a `NoSchedule` effect for
+[node conditions](/docs/concepts/scheduling-eviction/pod-eviction#node-conditions).
+The scheduler checks taints, not node conditions, when it makes scheduling
+decisions. This ensures that node conditions don't directly affect scheduling.
+You can ignore node conditions for newly created pods by adding the correct pod
+tolerations.
+
+For example, if the `DiskPressure` node condition is active, the control plane
+adds the `node.kubernetes.io/disk-pressure` taint and does not schedule new pods
+onto the affected node. If the `MemoryPressure` node condition is
```

Comment:
We should still say that the control plane adds the `node.kubernetes.io/memory-pressure` taint before talking about the toleration.

Something like:

If the MemoryPressure node condition is active, the control plane adds the node.kubernetes.io/memory-pressure taint. But it also adds a toleration....